account-compression/sdk/src/merkle-tree/index.ts

```diff
@@ -1,6 +1,6 @@
-import pkg from 'js-sha3';
-import * as Collections from 'typescript-collections';
-import { PublicKey } from '@solana/web3.js';
+import pkg from "js-sha3";
+import * as Collections from "typescript-collections";
+import { PublicKey } from "@solana/web3.js";
```

Comment:
nit: can you revert the `'` -> `"` changes? the rest of the repo uses `'`

Reply:
Fixed! Thanks